javacutil/src/org/checkerframework/javacutil/ErrorReporter.java

```diff
@@ -31,6 +31,21 @@ public static void errorAbort(String msg) {
         }
     }
 
+    /**
+     * Log an error message and abort processing.
+     * Call this method instead of raising an exception.
+     *
+     * @param msg the error message to log
```

Comment:
It would be good to document that in contrast to the above method this is now a format string.